espnet/nets/chainer_backend/asr_transformer/attention.py

```diff
@@ -0,0 +1,60 @@
+# encoding: utf-8
```

Comment:
This function must be at `espnet/espnet/nets/chainer_backend/`
How about `espnet/espnet/nets/chainer_backend/attention_transformer.py`?